### PR TITLE
add Intlim version 1.1.0

### PR DESCRIPTION
**Pull request by abotzki**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-intlim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
```
Comment (bgruening) [inline recipes/r-intlim/meta.yaml]: if this package needs to be compiled, can you please add compilers as well into the build section.

Comment (johanneskoester) [reply recipes/r-intlim/meta.yaml]: I think the compiler `{{ compiler('c') }}` should be explicitly added, right @bgruening?

Comment (abotzki) [reply recipes/r-intlim/meta.yaml]: Sorry to ask these maybe simple question: I've checked other comparable recipes and it seems that we could live without having any build step for Windows. The authors of the software indicate that there is no compilation necessary. I can confirm this on my Windows machine when installing it via devtools.
So, my idea was to completely remove the build step.

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**